src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -276,7 +276,7 @@ public static TracerSettings FromDefaultSources()
         /// AppSettings where available, and a local datadog.json file, if present.
         /// </summary>
         /// <returns>A new <see cref="IConfigurationSource"/> instance.</returns>
-        public static CompositeConfigurationSource CreateDefaultConfigurationSource()
+        public static IConfigurationSource CreateDefaultConfigurationSource()
```

Comment:
This is a public API change so we should flag it somehow, but I support this change

Reply:
Good catch, I'll revert this change.